### PR TITLE
add template functions system

### PR DESCRIPTION
**Pull request by SebastianPozoga**

### CODE REVIEW

File: services/template/funcs/main.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+	}
+	cuter := text[0:max]
+	last := strings.LastIndexAny(cuter, ". ")
+	if last > 0 {
+		return cuter[0:last-1] + "..."
+	}
+	return cuter + "..."
+}
+
+// Init inicjalize default teplate function
```
Comment (s3c0nDD) [inline services/template/funcs/main.go]: // Init initialize default template function
(typos)

File: services/template/main.go [hunk 2]
```diff
@@ -36,6 +37,12 @@ func (t *Template) Init(path string) error {
 	return nil
 }
 
+// Funcs adds the elements of the argument map to the template's function map.
```
Comment (s3c0nDD) [inline services/template/main.go]: // Funcs add elements of the argument map to the template's function map

Comment (SebastianPozoga) [inline services/template/main.go]: Well do you think a oryginal golang translation has typo?? 
"Funcs adds the elements of the argument map to the template's function map. It panics if a value in the map is ..."
https://golang.org/pkg/text/template/#Template.Funcs

It is present simple and you add "s" to third person always. ;-)

Comment (s3c0nDD) [inline services/template/main.go]: but a convention in this project is we do not say function does smth but function do something :P if we want change it we should change all other comments :no_entry:

**Pull request merged**